content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -44,6 +44,18 @@ to the name of the namespace. You can't change this label's value.
 This is useful if you want to target a specific namespace with a label
 {{< glossary_tooltip text="selector" term_id="selector" >}}.
 
+## kubernetes.io/limit-ranger
+
+Example: `kubernetes.io/limit-ranger: LimitRanger plugin set: cpu, memory request for container nginx; cpu, memory limit for container nginx`
```

Comment:
This example doesn't look like an "example" to me.

Reply:
The example for this annotation looks like this as it display -a request for limiting the resource was made and it was applied.

Reply:
I guess it should be clarified that this is not an annotation users should add, but an annotation that the system adds.

Reply:
yes it displays the status